### ISSUE TITLE
Web UI Client

### ISSUE THREAD

**Issue by bagelcafe** [smartscenes/multiscan#6]
Hi,

I collected the image data via iPad and upload these data to my private server which is based on your git code.
Based on the guide, i can configure the webui server to handle the image data from iPad.

Guide: https://3dlg-hcvc.github.io/multiscan/read-the-docs/web-ui/index.html#ui-overview

However, when I visit to my local page as http://localhost:3030/scans/manage to see the local server data. it shows "Loading" text and  it did not show anything.

![image](https://user-images.githubusercontent.com/35480129/208862476-c67b79da-322c-4271-86f9-75f02eaa1fa3.png)

I tried to debug web browser to see the error with dataTable.Editor as following.

manage-ajax.js 324 line Uncaught TypeError, $.fn.dataTable.Editor is not constructor 
manage:4

![image](https://user-images.githubusercontent.com/35480129/208866698-a4a1b720-eddb-4b94-a425-7d4a6a0c3db7.png)

Do I need to purchase licensed dataTable.Editor ?
Otherwise, can I avoid this error to see /scans/manage webpage?

Thank you for your help in advance.

Best Regards,

**Comment by SamMaoYS** [comment]
Hi @bagelcafe, it is great to hear that you scanned and uploaded data from iPad.

The [web-server](https://github.com/smartscenes/multiscan/tree/main/web-ui/web-server) is the backend for the web UI, to browse the uploaded data and visualize the reconstructed textured meshes, we use [web-client](https://github.com/smartscenes/multiscan/tree/main/web-ui/web-client), the frontend for the web UI.

We updated our [code and documentation](https://github.com/smartscenes/multiscan/tree/main/web-ui), so it will not need the vendor package (licensed dataTable.Editor) anymore. Please try out the updated web UI, thank you!

All the best.

**Comment by bagelcafe** [comment]
Hi @SamMaoYS ,

Thank you for your support and I updated web ui client and server codes via git system. 
After updating code, I can see local custom datasets from web UI client as following screen.

![image](https://user-images.githubusercontent.com/35480129/209595871-5f6ff203-b8bf-4406-b0af-9ee7c42bf73e.png)

From now on, I wanna to annotate one of our custom datasets to click annotate button and segmentation menu selected,
However, Segment Annotator webpages is trying to connect to sfu server as following screen shot.

![image](https://user-images.githubusercontent.com/35480129/209596222-8a9fb631-2863-41a3-a24d-7fe9eab2eae3.png)

Basically, index.vue file indicated sfu.ca server.

src="https://aspis.cmpt.sfu.ca/stk-multiscan/multiscan/grouped?annotate=true"></iframe>

How do I install and configure custom Segment Annotator on our server?

I appreciate your efforts and time for supporting this in advance.

Best Regards,

Jin.

**Comment by Orion520** [comment]
Thanks for the open source program. I have the same problem, how to solve it? Should I configure a local annotation server through SSTK?

**Comment by Orion520** [comment]
Hi! @SamMaoYS  
Thanks for the open source program. I have the same problem, how to solve it? Should I configure a local annotation server through SSTK?
